packages/docs-website/docs/app-devs/quick-start-dapp.md

```diff
@@ -67,3 +67,54 @@ This ephemeral key is only used for creating signatures that will be recovered i
 :::
 
 Our wallet uses the concept of a "Domain Budget" to further reduce the amount of user interaction that is required. The budget can be [approved](typescript-api/channel-client.channelclient.approvebudgetandfund) on the user's first visit to an application, and specifies a maximum send and maximum receive amount for each asset type (e.g. ETH). It then only needs to be administered or "topped-up" when the maximum amounts are reached. Otherwise, the wallet will not prompt the user at all when creating and closing channels, but will silently perform the necessary steps to do so. You can see budgets in action by trying out our [Web3Torrent](https://web3torrent.statechannels.org/) app.
+
+## Advanced: hosting a wallet locally
+
+There are some downsides to using the _hosted_ wallet with your app, particularly during development. It is possible to develop your Dapp against historical or future versions of our browser wallet, which also gives the flexibility to target a different Ethereum testnet or even a local blockchain.
+
+First, install the wallet as a devDependency in your project. You can even pin to a specific version.
+
+```shell
+yarn add --dev @statechannels/xstate-wallet
+```
+
+Next, run
+
+```shell
+cd node_modules/@statechannels/xstate-wallet && HUB_DESTINATION='0x0' yarn start
+```
+
+This will spin up a (i) (optionally) a local instance of [ganache](https://www.trufflesuite.com/docs/ganache/overview) which the [Nitro contracts](contract-api/contract-inheritance) are deployed to, and (ii) a webpack dev server for the wallet.
```

Comment:
```suggestion
This will spin up (i) (optionally) a local instance of [ganache](https://www.trufflesuite.com/docs/ganache/overview) which the [Nitro contracts](contract-api/contract-inheritance) are deployed to, and (ii) a webpack dev server for the wallet.
```